doc/authentication.md

```diff
@@ -23,7 +38,42 @@ The following example creates an API connection with a developer token:
 BoxAPIConnection api = new BoxAPIConnection("YOUR-DEVELOPER-TOKEN");
 ```
 
-### Normal Authentication
+### Server Authentication with JWT
+
+
+App Users allows your application to provision and control Box accounts that do not have an associated login
+and can only be accessed through the Content API by the controlling application. An App User is a Box account that 
+belongs to your Box Platform application and not an end-user of Box, like a [managed user](https://developer.box.com/v2.0/reference#user-object)
+It is important to emphasize that unlike typical Box accounts, these accounts do not have an associated login and can only be accessed through the Box API. 
+
+You may authenticate as the service 
+account to provision and mange users, or as an individual app user to make calls as that user. See the 
+[API documentation](https://github.com/box/box-node-sdk/blob/master/docs/authentication.md#app-user-authentication)
+for detailed instruction on how to use app auth. 
+
+App User client example: 
+```java
+BoxDeveloperEditionAPIConnection api = BoxDeveloperEditionAPIConnection.getAppUserConnection("USER-ID", "CLIENT-ID",
```

Comment:
We should explain what the encryption preference is, or give an enum of values for it